packages/static/src/subBanner/subBanner-description.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import CssInjection from '../utils/objectToCss/CssInjection'
+import styles from './styles/subBanner.module.css'
+
+interface Props {
+  children?: React.ReactNode
+  css?: unknown
+  className?: string
+}
+
+export type SubBannerDescriptionProps = Props &
+  Omit<React.HTMLAttributes<HTMLDivElement>, keyof Props>
+
+const SubBannerDescription = React.forwardRef<
+  HTMLDivElement,
+  SubBannerDescriptionProps
+>((props, ref) => {
+  const {children, css = {}, className = '', ...delegated} = props
+
+  return (
+    <CssInjection css={css} childrenRef={ref}>
+      <p
+        ref={ref}
+        {...delegated}
```

Comment:
we discussed about this. we don't call it 'delegated' anymore. it's htmpProps from now on